roles/openshift_facts/library/openshift_facts.py

```diff
@@ -114,6 +114,12 @@ def migrate_hosted_facts(facts):
             if 'router' not in facts['hosted']:
                 facts['hosted']['router'] = {}
             facts['hosted']['router']['selector'] = facts['master'].pop('router_selector')
+        if 'registry_selector' in facts['master']:
+            if 'hosted' not in facts:
+                facts['hosted'] = {}
+            if 'registry' not in facts['hosted']:
+                facts['hosted']['registry'] = {}
+            facts['hosted']['registry']['selector'] = facts['master'].pop('registry_selector')
```

Comment:
I actually am leaning towards @dgoodwin's pov here. We've discussed leveraging openshift_facts less for "desired state" and more for just retrieving info about the configured system instead. I'd be good with completely skipping openshift_facts here and potentially moving the default into a filter or lookup plugin to allow for re-use.